src/main/app/case/definition.ts

```diff
@@ -708,6 +708,8 @@ export interface CaseData {
   coDocumentsUploaded: ListValue<AdoptionDocument>[];
   coIsEverythingInPetitionTrue: YesOrNo;
   otherPlacementOrderType: string;
+  checkYourAnswersReturn: false;
```

Comment:
Have you added these two attribute in adoption-cos-api?

Reply:
Might not be needed if it is for only ui session. Please clarify.

Reply:
@ShakiraKP

Reply:
It should only be needed for the session so no I have not added them to cos-api